test/standalone/Vegetation/plant_hydraulics_test.jl

```diff
@@ -284,8 +284,8 @@ for FT in (Float32, Float64)
             )
             # Set system to hydrostatic equilibrium state by setting fluxes to zero, and setting LHS of both ODEs to 0
             function initial_compute_exp_tendency!(F, Y)
-                AI = (; leaf = LAI(0.0), root = RAI, stem = SAI)
-                T0A = FT(1e-8) * AI[:leaf]
+                AI = (; leaf = LAI(1.0), root = RAI, stem = SAI)
```

Comment:
Shouldn't this LAI vary spatially too? Why is it 1.0 everywhere?

Reply:
In this case, it is just a test at a single location, so it is OK to have it be constant. but we vary spatially in our global and regional runs!